plugins/jira-dashboard/package.json

```diff
@@ -83,5 +83,21 @@
   },
   "files": [
     "dist"
-  ]
+  ],
```

Comment:
I meant this is well? Never seen anything like this.

Reply:
Without it the JiraDashboardContent throws error because the config annotationPrefix value can't be found. I followed these docs when I added this config schema: https://backstage.io/docs/conf/defining/